### PR TITLE
Translations update

### PR DESCRIPTION
**Pull request by sikmir**
Issue #75

### CODE REVIEW

File: lang/gpxsee_fi.ts [hunk 1]
```diff
@@ -625,37 +601,37 @@
...
         <location filename="../src/GUI/gui.cpp" line="442"/>
         <source>&amp;Map</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Kartat</translation>
     </message>
     <message>
         <location filename="../src/GUI/gui.cpp" line="450"/>
         <source>&amp;Graph</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Kaaviokuva</translation>
```
Comment (tumic0) [inline lang/gpxsee_fi.ts]: Kartat and Kaaviokuva have the same shortcut letter. Every item in the same context should IMHO have a different letter.

Comment (sikmir) [reply lang/gpxsee_fi.ts]: Sure, updated.

File: lang/gpxsee_fi.ts [hunk 2]
```diff
@@ -625,37 +601,37 @@
...
         <location filename="../src/GUI/gui.cpp" line="471"/>
         <source>&amp;Data</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Tiedot</translation>
     </message>
     <message>
         <location filename="../src/GUI/gui.cpp" line="480"/>
         <source>&amp;Settings</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Asetukset</translation>
```
Comment (tumic0) [inline lang/gpxsee_fi.ts]: Now it's K&amp;aaviokuva and &amp;Asetukset that collide ;-)

Comment (sikmir) [reply lang/gpxsee_fi.ts]: Yeahh)) Updated again.

**Pull request merged**